CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
     * Move "Set password" page to be accessible via Admin Settings tab
   * Add success/fail messages to zones settings changes
   * Add state polling to some user workflows (stream starting, stopping; using presets) to remove the illusion of lag
+  * Sand off more rough edges too tiny to specifically list
```

Comment:
I think I'd prefer either explicit fixes listed here, a simple "UI fixes", or if they're too small omitting it from the CHANGELOG entirely. In this case, I like the second.

Reply:
Fair, I've just felt weird omitting like 7 PRs in a row from the changelog